.github/workflows/validate_packs.yml

```diff
@@ -21,6 +21,7 @@ jobs:
       - name: Validate files
         run: cd utils && yarn validate-quickstarts
 
```

Comment:
@tyreer so we would remove the `push` parameter above (github won't let me put the suggestion here) in order to make sure we always have info on what files were changed